spec/features/donuts/01_user_can_view_a_list_of_donuts_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "rails_helper"
+
+feature "user sees a list of donuts", %{
+   As a user
+   I want to be able to navigate to the donuts' page
+   To look at delicious donuts
+  } do
+  # ACCEPTANCE CRITERIA:
+  # Root is Donut#index
+  # See Links on Donut names
+
+  scenario "sees a list of donuts and a link for a new donut" do
+    donut_1 = Donut.create(
```

Comment:
^ address this hound violation by not assigning it to a variable.

Also, I would like to see these Donut objects be generated by FactoryGirl. I'll hop on over to you in person to explain this more. You can do it in another branch or in this one